gui-apps/swaync/swaync-0.7.3.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson vala gnome2-utils
+
+MY_PN="SwayNotificationCenter"
+DESCRIPTION="A simple notification daemon with a GTK gui for notifications and control center"
+HOMEPAGE="https://github.com/ErikReider/SwayNotificationCenter"
+SRC_URI="https://github.com/ErikReider/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+man scripting"
+
+DEPEND="
+	dev-libs/glib:2
+	dev-libs/gobject-introspection
+	dev-libs/json-glib
+	dev-libs/libgee:=
+	dev-libs/wayland
+	>=gui-libs/gtk-layer-shell-0.7.0[introspection]
+	gui-libs/libhandy:1
+	sys-apps/dbus
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:3
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	$(vala_depend)
+	man? ( app-text/scdoc )
+"
+
+src_prepare() {
+	default
+	vala_setup
+}
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use man man-pages)
+		$(meson_use scripting)
```

Comment:
I still feel like this should be unconditionally enabled, but as the actual package maintainer I'll leave the decision to you.

Some guidance from the devmanual - I think you could make a case either way:
https://devmanual.gentoo.org/general-concepts/use-flags/index.html#when-not-to-use-use-flags

Reply:
Please keep in mind that we don't always expose USE flags for every single option available in the build system.

It doesn't control any dependencies here and it leads to more combinations you need to test.

Your reasoning is fair, but it's also not something you can cite for all of these - it makes sense with moderation though. So, in this case, if you really feel it's worth keeping it optional, that's fine, but in general, you shouldn't map every option in the build system to a USE flag.

Reply:
Your arguments make sense, these USE flags have almost no impact on dependencies / file size / build time. Also, you are more experienced, I'm still learning how to do good ebuilds and make them "the Gentoo way", so as suggested by you guys I removed both USE flags. All features are now enabled unconditionally. Thank you for explaining this to me.